<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_Page</class>
 <widget class="QMainWindow" name="Home_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>344</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ỨNG DỤNG TÌM NGƯỜI MUỐN GẶP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="home_page_box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="title">
     <string>TRANG CHỦ</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>336</width>
       <height>173</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="title_label_1">
        <property name="font">
         <font>
          <family>Showcard Gothic</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 85, 255);
font: 12pt &quot;Showcard Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;WELCOME TO MY PROJECT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signup_button">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>ĐĂNG KÝ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signin_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ĐĂNG NHẬP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="credit_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>CREDIT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
